src/scss/vendors/_vendors.scss

```diff
@@ -55,7 +63,8 @@
 	}
 
 	button[aria-expanded="false"] {
```

Comment:
It would be easier to understand to replace `button` with `.fl-prefsEditor-showHide` since `button` gives an impression of both show/hide and reset buttons. As only the show/hide button has `aria-expanded` attribute, this definition only applies to it. Be more specific whenever possible.

Reply:
I'm also curious why you used both `margin-left` and `padding-left` at line 73, 75. As they have the same effect in this case, I wonder why you didn't just use one of them to reduce the complexity?

Reply:
I did this to have the grey border that separates "hide preferences" and "reset" centred when enhance inputs is turned on.

Reply:
> It would be easier to understand to replace `button` with `.fl-prefsEditor-showHide` since `button` gives an impression of both show/hide and reset buttons. As only the show/hide button has `aria-expanded` attribute, this definition only applies to it. Be more specific whenever possible.

This comment applies to elsewhere in _vendors.scss.

Reply:
bump. This comment is not fully addressed in _vendors.scss.

Reply:
The changes suggested in the first comment cause the errors mentioned [here](https://github.com/inclusive-design/wecount.inclusivedesign.ca/pull/337#issuecomment-663702549). Changing it back to how it was before.